src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -1739,7 +1739,7 @@ public ImmutableArray<DocumentId> GetDocumentIdsWithFilePath(string? filePath)
                 return ImmutableArray.Create<DocumentId>();
             }
 
-            return _filePathToDocumentIdsMap.TryGetValue(filePath, out var documentIds)
+            return _filePathToDocumentIdsMap.TryGetValue(filePath!, out var documentIds)
```

Comment:
is this part of the change?

Reply:
Yes.  Apparently the version of `string.IsNullOrEmpty` used here is not annotated.

---
In reply to: [303279493](https://github.com/dotnet/roslyn/pull/34916#discussion_r303279493) [](ancestors = 303279493)